app/assets/javascripts/table_floating_header.js

```diff
@@ -1,147 +1,147 @@
-var floating_header = function() {
+// var floating_header = function() {
```

Comment:
Why comment all this out instead of just pulling it?

Reply:
i was thinking about links needed to resolve but ill pull it